string binName
---
APCObject targetObject
APCObject[] binContents

